topics/transcriptomics/tutorials/de-novo/data-library.yaml

```diff
@@ -1,24 +1,52 @@
 ---
-libraries:
-    - name: de-novo
-      files:
-          - url: https://zenodo.org/record/254485/files/G1E_R1_forward_downsampled_SRR549355.fastqsanger.gz
-            file_type: "fastqsanger"
-          - url: https://zenodo.org/record/254485/files/G1E_R1_reverse_downsampled_SRR549355.fastqsanger.gz
-            file_type: "fastqsanger"
-          - url: https://zenodo.org/record/254485/files/G1E_R2_forward_downsampled_SRR549356.fastqsanger.gz
-            file_type: "fastqsanger"
-          - url: https://zenodo.org/record/254485/files/G1E_R2_reverse_downsampled_SRR549356.fastqsanger.gz
-            file_type: "fastqsanger"
-          - url: https://zenodo.org/record/254485/files/Megakaryocyte_R1_forward_downsampled_SRR549357.fastqsanger.gz
-            file_type: "fastqsanger"
-          - url: https://zenodo.org/record/254485/files/Megakaryocyte_R1_reverse_downsampled_SRR549357.fastqsanger.gz
-            file_type: "fastqsanger"
-          - url: https://zenodo.org/record/254485/files/Megakaryocyte_R2_forward_downsampled_SRR549358.fastqsanger.gz
-            file_type: "fastqsanger"
-          - url: https://zenodo.org/record/254485/files/Megakaryocyte_R2_reverse_downsampled_SRR549358.fastqsanger.gz
-            file_type: "fastqsanger"
-          - url: https://zenodo.org/record/254485/files/RefSeq_gene_annotations_mm10.bed.gtf.gz
-            file_type: "gtf"
-          - url: https://zenodo.org/record/254485/files/RNAseq_regions_of_interest.bed.gz
-            file_type: "bed"
+destination:
+  name: GTN - Material
+  type: library
+  description: Galaxy Training Network Material
+  synopsis: Galaxy Training Network Material. See https://training.galaxyproject.org
+items:
+- name: Transcriptomics
+  description: Training material for all kinds of transcriptomics analysis.
+  items:
+  - name: de-novo
+    items:
+    - url: https://zenodo.org/record/254485/files/G1E_R1_forward_downsampled_SRR549355.fastqsanger.gz
+      src: url
+      ext: fastqsanger
```

Comment:
fastqsanger.gz - this will prevent Galaxy from unpacking the file and wasting storage space.